doc/apidoc/smac.intensification.intensification.rst

```diff
@@ -0,0 +1,7 @@
+smac\.intensification\.intensification module
+=============================================
+
+.. automodule:: smac.intensification.intensification
```

Comment:
I would prefer to see all methods of a class. @KEggensperger @mlindauer what's your opinion on this?